scipy/linalg/flapack.pyf.src

```diff
@@ -735,11 +763,296 @@ interface
           :: lwork=2*minmn+MAX(maxmn,nrhs)
           ! check(lwork>=2*minmn+MAX(maxmn,nrhs))
```

Comment:
Ok, I found out that  there is already something similar to the gelss_lwork. The fortran source `linalg.src.calc_lwork.f`contain something similar.